src/components/Common/MainMenu.vue

```diff
@@ -89,6 +99,29 @@ export default {
   components: {
     EnvironmentSwitch
   },
+  data() {
+    return {
+      feedbackOptions: [
```

Comment:
Why factorizing the content of the dropdown in the component state? Is it meant to change dynamically based on user interaction?

Reply:
IMHO it allow a better readability also I think some data used for component rendering are totally legitimate to be there even if they are not dynamic
But I can just put it in the template if needed